tailwind.config.ts

```diff
@@ -139,7 +139,10 @@ export default {
           foreground: {
             DEFAULT: "hsl(var(--neutral-100))",
             secondary: "hsl(var(--neutral-50))",
-            inverse: "hsl(var(--neutral-0))",
+            inverse: {
+              DEFAULT: "hsl(var(--neutral-0))",
+              secondary: "hsl(var(--neutral-0) / 0.8)",
+            },
```

Comment:
It worries me that we keep adding "secondary" whenever we please. Is it too naive to try to solve this with opacity so we don't need to add extra colors, perhaps?